docs/StyleGuide/Style-Guide.md

```diff
@@ -293,15 +365,25 @@ Avoid too little or too much whitespace that reduces the overall readability.
 ```dafny
 // YES
 lemma MyLemma<A, B>(x: seq<seq<A>>, y: B) {
-    ...
+  ...
 }
 
 // NO
 lemma MyLemma <A,B> ( x : seq<seq<A>> , y :B){
-    ...
+  ...
 }
 ```
 
+#### Attributes
+Omit white space between the `:` and the attribute name.
+```dafny
+// YES
+method {:extern} m() { ... }
+
+// NO
+method {: extern } m() { ... }
```

Comment:
```suggestion
method {: extern} m() { ... }
```

Reply:
Can the space before "extern" also be removed? It's not a type ascription.

Reply:
The point is that the space before the extern is not recommended style, which is why the '// NO' on the line before.